testutil/testutil.go

```diff
@@ -19,25 +19,27 @@ var protocPath = func() string {
 	return "protoc"
 }
 
+// FileDescriptorSpec defines a specification for generating a FileDescriptorProto
 type FileDescriptorSpec struct {
+	// Filename is the output of the returned FileDescriptorProto.GetName().
 	Filename string
+	// Template defines a text/template to use for the proto source.
 	Template string
-	Data     interface{}
-	Deps     []*descriptorpb.FileDescriptorProto
+	// Data is plugged into the template to create the full source code.
+	Data interface{}
+	// Deps are any additional FileDescriptorProtos that the protocol compiler will need for the source.
+	Deps []*descriptorpb.FileDescriptorProto
```

Comment:
Maybe we should make it consistent to `protoc` arguments:
`Deps` -> `DescriptorSetIn` (`FileDescriptorSet`)
`AdditionalProtoPaths` -> `ProtoPaths`

Reply:
I considered it. One thing that's nice about this setup is that this function outputs FileDescriptorProtos already, so it's easy to do something like:
```
f1 := MustCreateFileDescriptorFromSpec(...)
f2 := MustCreateFileDescriptorFromSpec(FileDescriptorSpec{ ..., Deps: []*descriptorpb.FileDescriptorProto{f1}})
```